tests/har.spec.ts

```diff
@@ -276,18 +276,18 @@ it('should include sizes', async ({ contextFactory, server, asset }, testInfo) =
   expect(log.entries[1].request.url.endsWith('one-style.css')).toBe(true);
   expect(log.entries[1].response.bodySize).toBe(fs.statSync(asset('one-style.css')).size);
   expect(log.entries[1].response.headersSize).toBeGreaterThanOrEqual(100);
-  expect(log.entries[1].response._transferSize).toBeGreaterThanOrEqual(200);
+  expect(log.entries[1].response._transferSize).toBeGreaterThanOrEqual(150);
 });
 
 it('should work with gzip compression', async ({ contextFactory, server, browserName }, testInfo) => {
-  it.fixme(browserName === 'webkit');
+  it.fixme(browserName !== 'chromium');
   const { page, getLog } = await pageWithHar(contextFactory, testInfo);
-  server.enableGzip('/simple.json');
-  const response = await page.goto(server.PREFIX + '/simple.json');
+  server.enableGzip('/simplezip.json');
```

Comment:
Is the reasoning behind changing the file because it contains repeating stuff and gzip internally will shows the compression more directly?

Reply:
It was showing compression as -20, while it is >= in the spec, so I needed a positive compression.

Reply:
discussed offline: seemed like DevTools did it wrong.